cime_config/testdefs/testmods_dirs/clm/FatesSetupParamBuild/shell_commands

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Make sure the environment is setup to run the FATES modify parameter file tool
+
+# Function to check for errors and abort
+check_error () {
+   # Arguments: error, message
+   if [ $# != 2 ]; then
+      echo "Wrong number of arguments to check_error"
+   fi
+   error=$1
+   msg=$2
+   if [ $error != 0 ]; then
+      echo "ERROR: Command failed"
+      echo $msg
```

Comment:
Add `>&2` to send to `STDERR` instead of `STDOUT` (both lines here).

Reply:
Done